packages/terra-table/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 4.15.0 - (August 18, 2020)
+
 * Fixed
```

Comment:
theres two `fixed` tags on this change log

Reply:
Fixed: https://github.com/cerner/terra-core/pull/3131/commits/f80977ab8da3d91d0cd28c8644c8d17ade4ba7d8